SpatialGDK/Source/SpatialGDKEditor/Private/WorkerAssociationCustomization.cpp

```diff
@@ -0,0 +1,139 @@
+#include "WorkerAssociationCustomization.h"
+
+#include "SpatialGDK/public/SpatialGDKSettings.h"
+
+#include "PropertyEditor/Public/DetailLayoutBuilder.h"
+#include "PropertyEditor/Public/DetailWidgetRow.h"
+#include "PropertyEditor/Public/IDetailChildrenBuilder.h"
+#include "PropertyEditor/Public/PropertyCustomizationHelpers.h"
+#include "PropertyEditor/Public/PropertyHandle.h"
+#include "Widgets/SToolTip.h"
+#include "Widgets/Text/STextBlock.h"
+
+#define LOCTEXT_NAMESPACE "WorkerAssociationCustomization"
+
+FWorkerAssociationCustomization::FWorkerAssociationCustomization() : Settings(GetMutableDefault<USpatialGDKSettings>())
```

Comment:
can you just initializer list this in the header? not sure that's possible or desirable but seems better than empty constructor here?